packages/application-components/src/components/modal-pages/internals/modal-page-top-bar.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import { injectIntl } from 'react-intl';
+import {
+  Text,
+  CloseIcon,
+  FlatButton,
+  AngleLeftIcon,
+  SecondaryIconButton,
+  customProperties,
+} from '@commercetools-frontend/ui-kit';
+import messages from './messages';
+
+// Component to have a larger the clickable surface
+const LargeCloseIcon = props => (
+  <span
+    css={css`
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      &::after {
+        content: '';
+        position: absolute;
+        height: 35px;
+        width: 48px;
+        top: 0;
+        right: 0;
+        }
+    }`}
+  >
+    <CloseIcon size="medium" {...props} />
+  </span>
+);
+LargeCloseIcon.displayName = 'LargeCloseIcon';
+
+const ModalPageTopBar = props => (
+  <div
+    css={css`
+      position: relative;
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      padding: ${customProperties.spacingS} ${customProperties.spacingM};
+      background-color: ${props.color === 'gray'
+        ? customProperties.colorGray95
+        : customProperties.colorWhite};
+      border-bottom: 1px solid
+        ${props.color === 'gray'
+          ? customProperties.colorWhite
+          : customProperties.colorGray};
```

Comment:
Those tokens are deprecated, we should use the new ones: `colorSurface` and `colorNeutral`